### ISSUE TITLE
Build only works with internet connection

### ISSUE THREAD

**Issue by jo-m** [LUGGPublic/CG_Labs#2]
Hej!

`make` only works with an internet connection. An internet connection is needed obviously the first time to fetch dependencies, but should not be required after. However, the `assimp` dependency tries to fetch a git repo and thus fails:

```
[  2%] Performing update step for 'assimp'
[  4%] Performing configure step for 'assimp'
-- Looking for ZLIB...
-- Checking for module 'zzip-zlib-config'
--   No package 'zzip-zlib-config' found
-- Found ZLIB: optimized;/usr/lib/libz.dylib;debug;/usr/lib/libz.dylib
-- Checking for module 'minizip'
--   No package 'minizip' found
-- Build an import-only version of Assimp.
-- Enabled formats: 3DS AC ASE ASSBIN ASSXML B3D BVH COLLADA DXF CSM HMP IRRMESH IRR LWO LWS MD2 MD3 MD5 MDC MDL NFF NDO OFF OBJ OGRE OPENGEX PLY MS3D COB BLEND IFC XGL FBX Q3D Q3BS
P RAW SIB SMD STL TERRAGEN 3D X GLTF 3MF
-- Disabled formats:
-- Configuring done
CMake Warning (dev):
  Policy CMP0042 is not set: MACOSX_RPATH is enabled by default.  Run "cmake
  --help-policy CMP0042" for policy details.  Use the cmake_policy command to
  set the policy and suppress this warning.

  MACOSX_RPATH is not specified for the following targets:

   assimp

This warning is for project developers.  Use -Wno-dev to suppress it.

-- Generating done
-- Build files have been written to: /home/j/git/lund-cg/build/assimp/build
[  7%] Performing build step for 'assimp'
[ 79%] Built target assimp
[ 79%] Performing update step for 'gtest'
fatal: unable to access 'https://github.com/google/googletest.git/': Could not resolve host: github.com
CMake Error at /home/j/git/lund-cg/build/assimp/build/test/gtest/tmp/gtest-gitupdate.cmake:58 (message):
  Failed to fetch repository 'https://github.com/google/googletest.git'


make[5]: *** [test/gtest/src/gtest-stamp/gtest-update] Error 1
make[4]: *** [test/CMakeFiles/gtest.dir/all] Error 2
make[3]: *** [all] Error 2
make[2]: *** [assimp/src/assimp-stamp/assimp-build] Error 2
make[1]: *** [CMakeFiles/assimp.dir/all] Error 2
make: *** [all] Error 2
```

**Comment by pierremoreau** [comment]
I think this should be already fixed by e952fa74637d4791b0904a8deb555f194146a453. There is no need to build assimp’s test suite, and I didn’t wanted to, but I made a typo when disabling it.

**Comment by jo-m** [comment]
Confirmed this after merge. Build is now also faster.
